### ISSUE TITLE
SSL Client Authentication with mosca broker

### ISSUE THREAD

**Issue by Yuu94** [mcollina/mosca#677]
Hello.
I want to perform SSL client authentication between mosca and IoT device.
Is it possible now?

**Comment by mcollina** [comment]
Yes you can, you just have to supply the correct key, certificate and ca on both sides. I would recommend experimenting just with barebone tls module, and then do it here.

**Comment by srinivasyl** [comment]
@mcollina from where i can generate certificates for client and server?
